build.sbt

```
@@ -550,11 +550,22 @@ lazy val `parser-service` = (project in file("lib/scala/parser-service"))
 lazy val `docs-generator` = (project in file("lib/scala/docs-generator"))
   .dependsOn(syntax.jvm)
   .dependsOn(cli)
+  .configs(Benchmark)
   .settings(
     libraryDependencies ++= Seq(
       "commons-cli" % "commons-cli" % commonsCliVersion
     ),
-    mainClass := Some("org.enso.docs.generator.Main")
+    mainClass := Some("org.enso.docs.generator.Main"),
+    inConfig(Benchmark)(Defaults.testSettings),
+    Benchmark / unmanagedSourceDirectories +=
+      baseDirectory.value.getParentFile / "bench/scala",
```

```suggestion
      baseDirectory.value.getParentFile / "bench" / "scala",
```

A very minor nitpick, but I think you should use the separator operator so that it uses the correct separator on all platforms (in practice it shouldn't cause problems, but better to be sure).